integrationtests/src/test/java/org/axonframework/integrationtests/eventsourcing/eventstore/jdbc/Oracle11EventTableFactoryTest.java

```diff
@@ -18,32 +18,43 @@
 
 import org.axonframework.eventsourcing.eventstore.jdbc.EventSchema;
 import org.axonframework.eventsourcing.eventstore.jdbc.Oracle11EventTableFactory;
-import org.junit.jupiter.api.AfterEach;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
-import org.opentest4j.TestAbortedException;
+import org.junit.jupiter.api.*;
+import org.testcontainers.containers.OracleContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
 
 import java.sql.Connection;
 import java.sql.DriverManager;
 import java.sql.SQLException;
 
 import static org.axonframework.common.io.IOUtils.closeQuietly;
 
+/**
+ * Integration test class validating the {@link Oracle11EventTableFactory}.
+ *
+ * @author Joris van der Kallen
+ */
+@SuppressWarnings({"SqlDialectInspection", "SqlNoDataSourceInspection"})
+@Testcontainers
 class Oracle11EventTableFactoryTest {
 
+    private static final String USERNAME = "system";
+    private static final String PASSWORD = "oracle";
+
+    @Container
+    private static final OracleContainer ORACLE_CONTAINER = new OracleContainer("gautamsaggar/oracle11g:v2")
+            //Disable oracle.jdbc.timezoneAsRegion as when on true GHA fails to run this test due to missing region-info
+            .withEnv("oracle.jdbc.timezoneAsRegion", "false");
```

Comment:
You have to set it on the connection (few lines below, on `setUp`)

Reply:
If you say so. Kinda shitty to test locally, as it simply works there... Would like some resources where you've found that bit, as my Oracle knowledge doesn't extend that far tbh.

Reply:
Pushed, let's see if you're right ;-)

Reply:
It seems you were, bravo